apps/web/pages/auth/forgot-password/[id].tsx

```diff
@@ -74,6 +76,14 @@ export default function Page({ requestId, isRequestExpired, csrfToken }: Props)
       </>
     );
   };
+  useEffect(() => {
+    // disbale Reset Password button if password input field is empty
+    if (passwordValue) {
+      setIsEmpty(passwordValue.length === 0);
+      return;
+    }
+    setIsEmpty(true);
+  }, [passwordValue]);
```

Comment:
why are using setState and useEffect here ?
just use 

```suggestion
const isEmpty = passwordValue.length === 0
```